controllers/csm_controller.go

```diff
@@ -93,7 +93,7 @@ const (
 	CSMFinalizerName = "finalizer.dell.emc.com"
 
 	// CSMVersion -
-	CSMVersion = "v1.10.0"
+	CSMVersion = "v2.0.0-alpha"
```

Comment:
If we're using this version to reconcile authorization, we should create a new variable. Otherwise, lets keep at v1.10.0 to not break any driver operations

Reply:
Yeah, seems not for reconciliation, my bad. I thought it was used in the paths, but looks like it works with v1.10.0 as well. reverted back to v1.10.0